docs/extend/extend-apiml/api-mediation-onboarding-with-spring-boot-enabler

```diff
@@ -0,0 +1,502 @@
+# Onboarding a Spring Boot based REST API Service
+
+This guide is part of a series of guides to onboard a REST API service with the Zowe API Mediation Layer.
+As an API developer, you can onboard your REST API service built with the Spring Boot framework with the Zowe API Mediation Layer.
+
+**Note:** Before version 1.12, the API ML provided an integration enabler based on Spring Cloud Netflix components. From version 1.12 and later, the enabler has been replaced with new implimentation based on the Plain Java Enabler (PJE) that is not backwards compatiable with the previous enabler versions.
```

Comment:
maybe link for Plain Java Enabler (PJE) ?

Reply:
Link to the PJE documentation is provided at line 11 close to the beginning of the file.